crates/blockchain-tree/src/blockchain_tree.rs

```diff
@@ -1032,7 +1032,7 @@ impl<DB: Database, C: Consensus, EF: ExecutorFactory> BlockchainTree<DB, C, EF>
 
         let tip = tx.tip_number()?;
         let revert_range = (revert_until + 1)..=tip;
-        info!(target: "blockchain_tree", "Revert canonical chain range: {:?}", revert_range);
+        info!(target: "blockchain_tree", "Unwinding canonical chain blocks: {:?}", revert_range);
```

Comment:
nit: it's not clear to me when we choose to use a separate log field, and when to include the field into the message itself